docs/src/prototypes/ChatPeoplePicker/ChatPeoplePickerExample.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react'
+import ChatPeoplePicker from './ChatPeoplePicker'
+
+const items = [
+  {
```

Comment:
Adding more items causes the opened element to be cut in the middle of some item. Maybe we should consider having some max heights on the element displaying the items in the dropdown.
![image](https://user-images.githubusercontent.com/4512430/47006552-753a6300-d136-11e8-8f0f-796c42b9b7e3.png)

Reply:
added default maxWidth and width. Also the prototype will accept these as props.